src/odemis/util/img.py

```diff
@@ -1289,3 +1288,62 @@ def max_intensity_projection(data: model.DataArray, axis: int = 0) -> model.Data
     md = data.metadata.copy()
     md[model.MD_DIMS] = "YX"
     return model.DataArray(proj, md)
+
+
+def get_merged_raw_image(das: model.DataArrayShadow, z: int) -> model.DataArray:
+    """
+    Returns the entire raw data of DataArrayShadow at a given zoom level
+    das (DataArrayShadow): shadow of the raw data
```

Comment:
I feel like also putting the type in the docstring while utilizing Python's typing mechanism is double the work and prone to desync between docstring and function definition over time.

Reply:
Indeed, the new "style" is to only put the types as hint, not in the docstring. Here, Patrick "upgraded" the function to include type hints, so he could drop the ones from the docstring.

Reply:
removed